src/graphics/axis.hpp

```diff
@@ -0,0 +1,95 @@
+#ifndef AXIS_HPP
+#define AXIS_HPP
+
+#include "drawable.hpp"
+#include "esthetic.hpp"
+
+#include <SFML/Graphics/CircleShape.hpp>
+#include <SFML/Graphics/RectangleShape.hpp>
+
+namespace graphic {
+
+class Axis : public Drawable {
+
+  private:
+    sf::RectangleShape line;
+    sf::CircleShape    arrow;
+
+    float        size_arrow;
+    sf::Vector2f size_line;
+    sf::Vector2f position_line;
+    sf::Vector2f position_arrow;
+    int          orientation;
+
+    void fill_variables(Coordinate c);
+    void create() override;
```

Comment:
I would rename as `init ()` and use `create ()` functions for what actually build an object (not just initialize the attributes)

Reply:
Right away my lord